test/test_e2e.py

```diff
@@ -0,0 +1,337 @@
+import unittest
+from unittest.case import SkipTest
+from SBTi.interfaces import (
+    EScope,
+    ETimeFrames,
+    IDataProviderCompany,
+    IDataProviderTarget,
+    PortfolioCompany,
+)
+from SBTi.target_validation import TargetProtocol
+from SBTi.temperature_score import EngagementType, Scenario, ScenarioType, TemperatureScore
+from SBTi.data.data_provider import DataProvider
+from SBTi.portfolio_aggregation import PortfolioAggregationMethod
+import copy
+
+import SBTi
+from typing import List
+
+
+class TestDataProvider(DataProvider):
```

Comment:
maybe good to move this class to a separate file that it's easier to re-use (and find) it.

Reply:
Yeah, actually tried this but reversed it since nose is playing around with the sys.path therefore an import from a test/utils.py for example throws errors (https://stackoverflow.com/questions/16200333/import-errors-when-running-nosetests-that-i-cant-reproduce-outside-of-nose) so i kept it simple for now since i did not want to include in the the SBTi module.